cpp/ql/test/library-tests/ir/ssa/aliased_ssa_ir.expected

```diff
@@ -639,7 +639,7 @@ ssa.cpp:
 #-----|   Goto -> Block 3
 
 #  153|   Block 3
-#  153|     m3_0(Point)        = Phi                : from 1:~m1_5, from 2:m2_3
+#  153|     m3_0(Point)        = Phi                : from 1:m1_5, from 2:m2_3
```

Comment:
This change LGTM, but the comment on `ssa.cpp:153` says:
```
// Only one reaching def must exactly overlap, but we should still get a Phi for it.
```

Both of the reaching defs now exactly overlap (the inexactness is handled by a chi instruction, not this phi instruction), so I find the comment to be misleading. Or did I misunderstand it?

Reply:
I’ve attempted to clarify the comment.